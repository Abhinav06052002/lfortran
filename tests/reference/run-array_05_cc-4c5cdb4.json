{
    "basename": "run-array_05_cc-4c5cdb4",
    "cmd": "lfortran --continue-compilation --no-color {infile}",
    "infile": "tests/errors/array_05_cc.f90",
    "infile_hash": "9c6af3f3a5224c50b662958d2d3f1acb41def92f2b8873cfff2633ac",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "run-array_05_cc-4c5cdb4.stdout",
    "stdout_hash": "a779ee4f7341ddce58a41cee3d249ecf1b37216d7d4f16eb4e0f528e",
    "stderr": "run-array_05_cc-4c5cdb4.stderr",
    "stderr_hash": "aa3b3cd03fe13f861e1ee49e6abed92aab3c9e567193dd39d91505b4",
    "returncode": 0
}