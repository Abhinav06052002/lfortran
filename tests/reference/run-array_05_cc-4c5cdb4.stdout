(TranslationUnit
    (SymbolTable
        1
        {
            array_05_cc:
                (Program
                    (SymbolTable
                        4
                        {
                            t:
                                (ExternalSymbol
                                    4
                                    t
                                    2 t
                                    module_array_05
                                    []
                                    t
                                    Public
                                ),
                            type_01:
                                (Variable
                                    4
                                    type_01
                                    []
                                    Local
                                    ()
                                    ()
                                    Default
                                    (StructType
                                        4 t
                                    )
                                    ()
                                    Source
                                    Public
                                    Required
                                    .false.
                                )
                        })
                    array_05_cc
                    [module_array_05]
                    [(Print
                        (StringConstant
                            "compilation continued despite errors"
                            (String 1 36 () PointerString)
                        )
                    )]
                ),
            module_array_05:
                (Module
                    (SymbolTable
                        2
                        {
                            t:
                                (Struct
                                    (SymbolTable
                                        3
                                        {
                                            x:
                                                (Variable
                                                    3
                                                    x
                                                    []
                                                    Local
                                                    ()
                                                    ()
                                                    Default
                                                    (Allocatable
                                                        (Array
                                                            (Real 4)
                                                            [(()
                                                            ())]
                                                            DescriptorArray
                                                        )
                                                    )
                                                    ()
                                                    Source
                                                    Public
                                                    Required
                                                    .false.
                                                )
                                        })
                                    t
                                    []
                                    [x]
                                    Source
                                    Public
                                    .false.
                                    .false.
                                    []
                                    ()
                                    ()
                                )
                        })
                    module_array_05
                    []
                    .false.
                    .false.
                )
        })
    []
)
